app/views/people/_current_roles.html.haml

```diff
@@ -1,36 +1,25 @@
 %ul.list--pipe
-  - @history[:current].each do |role|
-    %li
-      .list--details
-        - if Grom::Helper.get_id(role.type) == 'SeatIncumbency'
-          %h3.content--xs
-            = t('.parliamentary_role').capitalize
-            - if role.class == GroupingHelper::GroupedObject
-              %h4.lead
-                = t('.mp_for')
-                = link_to role.nodes[0].constituency.name, constituency_path(role.nodes[0].constituency.graph_id)
-              %p= "Elected #{role.nodes.count} times"
-              %ul.list--bullet
-                - role.nodes.each do |node|
-                  %li= "#{node.date_range}"
-            - else
-              %h4.lead
-                = t('.mp_for')
-                = link_to role.constituency.name, constituency_path(role.constituency.graph_id)
-              %p= "#{role.date_range}"
+  - if current_roles['GovernmentIncumbency']
+    - current_roles['GovernmentIncumbency'].each do |role|
```

Comment:
These above lines could be consolidated if current_roles is a hash with: `current_roles.fetch('GovernmentIncumbency', []).each do |role|`